lib/sidekiq/pool/cli.rb

```diff
@@ -5,6 +5,8 @@ module Sidekiq
   module Pool
     class CLI < Sidekiq::CLI
       def initialize
+        @settings = YAML.load_file('config/sidekiq-pool.yml')
```

Comment:
Maybe pool config file path could be passed by optional param with default to `config/sidekiq-pool.yml`?